src/core/public/_variables.scss

```diff
@@ -1,3 +1,8 @@
 @import '@elastic/eui/src/global_styling/variables/header';
 
+// height of the header banner
+$kbnHeaderBannerHeight: $euiHeaderHeightCompensation * 1.1;
```

Comment:
I'm wondering about this as hardcoded vs configurable. The header is fairly thick, and combined (eventually) with an equal-sized footer, will take up more screen than may be desired. The banners I typically see are as thin as possible.

Would it be difficult to make this a user configurable item w/ the current height as the default? Maybe for user-simplicity it's `['default', 'compact']`, where `compact` puts just a few px of padding around the text. Or, just accept `px` values?

Reply:
> I'm wondering about this as hardcoded vs configurable

Fully configurable and/or dynamic values are just not technically doable with the fixed / static layout we are currently using in Kibana. Maybe after https://github.com/elastic/kibana/issues/86528, but definitely not now.

> Would it be difficult to make this a user configurable item w/ the current height as the default? Maybe for user-simplicity it's ['default', 'compact']

We could, but every layout option just duplicates all the stylings and mixins we will have to use. I would really avoid that if possible, and would also wait until https://github.com/elastic/kibana/issues/86528 ease such integration

Reply:
+1 to this approach: waiting for the page level service for true customization and decreasing the default height for now. It's currently calculating to `53.9px`, but I think we could get by with something smaller like `40px` (i.e. `$euiSizeXXL`)

Reply:
Makes sense.

FWIW, [the plugin](https://github.com/elastic/fed-sa/blob/master/kibana_banner/public/plugin.ts#L32) we've been distributing to customers uses a height of `24px`, so there is definitely room to shrink. 40 > 53.9 :)